projects/geo/src/lib/print/shared/print.service.ts

```diff
@@ -340,6 +342,8 @@ export class PrintService {
         try {
           this.addCanvas(doc, canvas, size, margins);
         } catch (err) {
+          alert('2' + err.toString());
```

Comment:
Is this intended or you forgot to remove it?

Reply:
Forgot, just removed it